pkg/agent/openflow/network_policy.go

```diff
@@ -1734,6 +1734,8 @@ type featureNetworkPolicy struct {
 	ovsMetersAreSupported bool
 	enableDenyTracking    bool
 	enableAntreaPolicy    bool
+	connectUplinkToBridge bool
```

Comment:
I think this can be remove since it is not used anywhere except when initializing.

Reply:
Thanks. Removed this.